### PR TITLE
pkgng: Add stdout and stderr to response object

### PR DESCRIPTION
**Pull request by dsg22**
To ease debugging if something goes wrong during pkg run.

##### SUMMARY
Expose the stdout and stderr of pkg through the result object of the pkgng module. Currently, if something goes wrong during the pkg run, the user has no way to see what happened, which makes recovering from that situation more difficult.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
pkgng

### CODE REVIEW

**Comment by felixfontein** [comment]
You need a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

**Comment by dsg22** [comment]
Thanks @felixfontein - added a changelog fragment.

File: changelogs/fragments/560-pkgng-add-stdout-and-stderr.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - pkgng - added stdout and stderr attributes to the result (https://github.com/ansible-collections/community.general/pull/560)
```
Comment (felixfontein) [inline changelogs/fragments/560-pkgng-add-stdout-and-stderr.yaml:2]:
```suggestion
  - pkgng - added ``stdout`` and ``stderr`` attributes to the result (https://github.com/ansible-collections/community.general/pull/560).
```

**Comment by dsg22** [comment]
> I assume you tested this?

There aren't really any automated tests for this module at present. I manually tested package installs (state: present), upgrades (state: latest with an older package installed), uninstalls (state: absent) and autoremove. They all work as expected.

**Comment by felixfontein** [comment]
@dsg22 thanks for implementing this!

**Pull request merged**